_data/workshops/2025-03-11-scikit-learn-mooc.yml

```diff
@@ -0,0 +1,7 @@
+_id: 88bbe640-cc45-11ef-8eb0-a7b04ae4ec7b
+country: NL
+startdate: '2025-03-11'
+enddate: '2025-03-12'
+lesson_title: Machine Learning in Python with Scikit-learn
+lesson_slug: scikit-learn-mooc
```

Comment:
@svenvanderburg is this one of your workshops? Or was it submitted by a colleague?

The `lesson_slug` field should contain the slug of the lesson repository name (e.g. `deep-learning-intro` for https://github.com/carpentries-incubator/deep-learning-intro) but there is no `scikit-learn-mooc` lesson in the Incubator (yet?)

Reply:
Yeah, this is from my team. This refers to https://github.com/esciencecenter-digital-skills/scikit-learn-mooc (which is indeed not in the incubator since it is not in carpentries format). I will inform my team members to only include carpentries incubator lessons. Sorry!